src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1508,12 +1516,162 @@ void invalidMultipleHandles() {
             @Test
             // One handle on entity cluster in hypothesis
             void valid() {
-                makeHypothesis(model, getUri("hypothesis-1"), Collections.singleton(entity), system);
-                testValid("NISTHypothesis.invalid: Each entity cluster in the hypothesis graph must have " +
+                markImportance(makeHypothesis(model, getUri("hypothesis-1"),
+                        Collections.singleton(entity), system), 100);
+                testValid("NISTHypothesis.valid: Each entity cluster in the hypothesis graph must have " +
                         "exactly one handle");
             }
         }
 
+        // Each hypothesis graph must have exactly one hypothesis importance value
+        @Nested
+        class HypothesisImportanceValue {
+            @Test
+            void invalid() {
+                //invalid hypothesis, no importance value
+                makeHypothesis(model, getUri("hypothesis-1"), Collections.singleton(entity), system);
+                testInvalid("NISTHypothesis.invalid: Each hypothesis graph must have exactly one" +
+                        " hypothesis importance value");
+            }
+            @Test
+            void valid() {
+                markImportance(makeHypothesis(model, getUri("hypothesis-1"),
+                        Collections.singleton(entity), system), 100);
+                testValid("NISTHypothesis.valid: Each hypothesis graph must have exactly one" +
+                        " hypothesis importance value");
+            }
+        }
+
+        // Each event or relation (cluster) in the hypothesis must have exactly one importance value
+        @Nested
+        class HypothesisEntityRelationClusterImportanceValue {
+
+            private final String documentEventUri = getUri("ldc:event-1");
```

Comment:
Get tacks the string on the end of http://www...ldc#. You probably don't want to use ldc:, etc.

Reply:
ah okay, I was trying to make it look similar to the example. I'll remove the ldc: portion.